TravisCI fails on Linux

### Summary
Currently Travis runs are failing on Trusty because of this:

```
CYCLO     go: downloading github.com/fzipp/gocyclo v0.3.1
go: github.com/fzipp/gocyclo upgrade => v0.3.1
/bin/sh: 1: gocyclo: not found
make: *** [codequality] Error 1
The command "make travis" exited with 2.
```

--- a/Makefile
+++ b/Makefile
@@ -192,7 +192,7 @@ codequality:
 
 	@echo -n "     CYCLO     "
 	@which gocyclo > /dev/null; if [ $$? -ne 0 ]; then \
-		$(GO) get -u github.com/fzipp/gocyclo; \
+		$(GO) get -u github.com/fzipp/gocyclo/cmd/gocyclo; \
 	fi
 	@$(foreach gofile, $(GOFILES_NOVENDOR),\
 			gocyclo -over 22 $(gofile) || exit 1;)

